.vscode/settings.json

```diff
@@ -6,21 +6,16 @@
         "**/Thumbs.db": true,
         ".idea": true,
         ".metadata": true,
-        "Build": true,
-        "Instrumented": true,
-        "node_modules": true,
         ".settings": true,
         ".externalToolBuilders": true,
         ".project": true,
-        "launches": true,
-        "Cesium-*.zip": true,
-        "cesium-*.tgz": true,
-        "Apps/Sandcastle/jsHintOptions.js": true,
-        "Apps/Sandcastle/gallery/gallery-index.js": true,
-        "Source/Cesium.js": true,
-        "Source/Shaders/**/*.js": true,
-        "Specs/SpecList.js": true,
-        "npm-debug.log": true
+        "launches": true
+    },
+    "search.exclude": {
+        "Build": true,
+        "Instrumented": true,
+        "node_modules": true,
+        "Source/Shaders/**/*.js": true
```

Comment:
Are you sure you want these converted js files to show in the directory tree?  Seems like they could get in the way when searching for actual shader code.

Reply:
If you think it will be an issue, we can add them to the search exclude too.  I would like them to be in the tree though.

Reply:
Certainly they should be excluded from search, as they contain copies of all shader code that might be searched for.  Thanks.

Reply:
Brain fart.  The link you're commenting on is already excluding them from the search, so this should be good to go.